README.md

```diff
@@ -0,0 +1,106 @@
+OpenIddict<br><sub>The OpenID Connect server you'll be addict to.</sub>
+=======
+
+AppVeyor: [![Build status](https://ci.appveyor.com/api/projects/status/d0d8git3o6lqkvbm?svg=true)](https://ci.appveyor.com/project/aspnet-contrib/core)
+
+Travis: [![Build status](https://travis-ci.org/openiddict/core.svg)](https://travis-ci.org/openiddict/core)
+
```

Comment:
FYI, there's no official release on NuGet.org yet. I'd really like to upload an alpha version for RC1 though.